examples/src/main/java/nl/jcore/java8demo/streaming/StreamingExecutionPlanExample.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         final List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8);
         final List<String> output = numbers.stream()
                 .filter(i -> i % 2 == 0)
-                .sorted((i1, i2) -> i2.compareTo(i1))
+                .sorted((i1, i2) -> i2.compareTo(i1)) // Alternative: .sorted(Comparator.reverseOrder())
```

Comment:
The alternative depends on the input list. However, this is a streaming example, so not sure which is more descriptive (show strength of lambda vs. simplicity of static factory).